opendbc/car/docs/generate_docs.py

```diff
@@ -1,15 +1,24 @@
+#!/usr/bin/env python3
+import argparse
 from collections import defaultdict
 import jinja2
+import os
 from enum import Enum
 from natsort import natsorted
 
 from opendbc.car import gen_empty_fingerprint
+from opendbc.car.common.basedir import BASEDIR
 from opendbc.car.structs import CarParams
-from opendbc.car.docs_definitions import CarDocs, Column, CommonFootnote, PartType
+from opendbc.car.docs.definitions import CarDocs, Column, CommonFootnote, PartType
 from opendbc.car.car_helpers import interfaces, get_interface_attr
 from opendbc.car.values import PLATFORMS
 
 
+# FIXME: make sure we run in a sensible way if not checked out as a submodule, try-except for writing to CARS.md
+CARS_MD_OUT = os.path.join(BASEDIR, "../", "../", "../", "docs", "CARS.md")
```

Comment:
The path to the out file should be in openpilot and passed in

Reply:
> The path to the out file should be in openpilot and passed in.

I can do that.

> Since we don't write any docs in opendbc, probably best to leave everything in openpilot for now

We will. The new ALL-CARS.md (or whatever name we bikeshed) will live in opendbc. It needs to run, generate, save, and test in opendbc without having openpilot available.

CARS.md only lives in openpilot because we don't want to change the link until we have a bigger step-change in how it looks and works.

> from opendbc.car.docs import generate_cars_md, get_all_car_docs

I'll see if I can juggle things to maintain that import location.

Reply:
You can just make docs.py executable and write the opendbc docs by default, then openpilot's docs.py can stay